src/trusted/service_runtime/nacl_syscall_common.h

```diff
@@ -301,6 +306,14 @@ int32_t NaClSysTestInfoLeak(struct NaClAppThread *natp);
 
 int32_t NaClSysTestCrash(struct NaClAppThread *natp, int crash_type);
 
+int32_t NaClSysPipe(struct NaClAppThread  *natp, uint32_t *pipedes);
+int32_t NaClSysFork(struct NaClAppThread  *natp);
+int32_t NaClSysExecv(struct NaClAppThread  *natp);
+int32_t NaClSysExecve(struct NaClAppThread  *natp, void* path, void* argv, void* envp);
```

Comment:
Style consistency for pointer names (`... *natp` vs `void* ...`).

Reply:
Ah, good catch. Missed this in my original revision, will fix.